### ISSUE TITLE
disallowSpaceAfterObjectKeys needs special handling for computed properties

### ISSUE THREAD

**Issue by iamstarkov** [jscs-dev/node-jscs#1742]
```
disallowSpaceAfterObjectKeys: Illegal space after key at ./src/Nav.jsx :
    17 |
    18 |    const classes = classNames({
    19 |      [block + '--default']: this.props.navStyle === 'default',
----------------------^
    20 |      [block + '--primary']: this.props.navStyle === 'primary',
    21 |      [block + '--success']: this.props.navStyle === 'success',
```

it seems wrong to me. Im using [nordnet preset, which extend airbnb preset](https://github.com/nordnet/jscs-config-nordnet/blob/master/lib/presets/nordnet-preset.json#L3). it’s legal and should be allowed because it aligned with everything else in codestyle

**Comment by hzoo** [comment]
Made a fix before that obviously didn't work :stuck_out_tongue: https://github.com/jscs-dev/node-jscs/pull/1481/files. Will do a PR

**Comment by iamstarkov** [comment]
as far as #1481 merged, then this issue is also can be closed by 6bbf569

**Comment by hzoo** [comment]
https://github.com/jscs-dev/node-jscs/blob/master/lib/rules/disallow-space-after-object-keys.js#L143

I just did `var keyToken = file.getLastNodeToken(property.key);` for your particular case but I think I want to create more test cases? Although I think it should work.
